Pipfile

```diff
@@ -4,7 +4,7 @@ url = "https://pypi.org/simple"
 verify_ssl = true
 
 [dev-packages]
-mypy = "*"
+mypy = "==0.780"
```

Comment:
i upgraded to the latest mypy version on `main` and it didn't cause any problems. do you remember what happened when you tried?

Reply:
oh weird i made a new branch to check and it does fail on CI: https://app.circleci.com/pipelines/github/votingworks/arlo/3144/workflows/4410b01b-67c4-4273-8c44-1297e83a0f04/jobs/4966

Reply:
It was complaining about types in `server/util/group_by.py` and somewhere else. See here: https://app.circleci.com/pipelines/github/votingworks/arlo/3137/workflows/3be6d301-e3cf-46c3-aa56-8eb77e560549/jobs/4936

Reply:
fixing this on #849

Reply:
so you can take this out